internal/daemon/controller/internal/metric/cluster.go

```diff
@@ -33,6 +34,17 @@ var grpcRequestLatency prometheus.ObserverVec = prometheus.NewHistogramVec(
 	metric.ListGrpcLabels,
 )
 
+// activeConnections keeps a count of the current active connections to a controller.
+var activeConnections = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: globals.MetricNamespace,
+		Subsystem: clusterSubSystem,
+		Name:      "active_connections",
```

Comment:
updated the metric name to "active_network_connections"

Reply:
actually I don't think I should change the name because it would be inconsistent with the existing proxy websocket gauge metric name `active_connections`. Perhaps I just call it out in the Help description?